js/lesson-2.js

```diff
@@ -0,0 +1,147 @@
+/*
+var a = 1, b = 1, c, d;
+c = ++a; alert(c);           // 2 Сначала увеличение на 1 потом присваивание
+d = b++; alert(d);           // 1 b увеличится на 1 после присваивания
+c = (2+ ++a); alert(c);      // 5 a = 2 увеличивается на 1 + 2 затем присваивание
+d = (2+ b++); alert(d);      // 4 b увеличится на 1 после присваивания
+alert(a);                    // 3 Изначально a равно 1 затем два раза увеличение на 1
+alert(b);                    // 3 Изначально b равно 1 затем два раза увеличение на 1
+*/
+
+
+/*
+var a = 2;
+var x = 1 + (a *= 2);
+5
+*/
+
+
+/*Объявить две целочисленные переменные — a и b и задать им произвольные начальные значения. Затем написать скрипт, который работает по следующему принципу:
+если a и b положительные, вывести их разность;
+если а и b отрицательные, вывести их произведение;
+если а и b разных знаков, вывести их сумму;
+Ноль можно считать положительным числом.*/
+
+/*
+var a = getRandomInt(-100, 100);
+var b = getRandomInt(-100, 100);
+
+function getRandomInt(min, max) {
+    return Math.floor(Math.random() * (max - min)) + min;
+}
+
+if (a >= 0 && b >= 0) {
+    alert("Разность равна: " + (a - b));
+} else if (a < 0 && b < 0) {
+    alert("Произведение равно: " + (a * b));
+} else if (a < 0 && b >= 0 || a >= 0 && b < 0) {
```

Comment:
Здесь можно обойтись просто else
```suggestion
} else {
```